docs/csharp/programming-guide/events/how-to-use-a-dictionary-to-store-event-instances.md

```diff
@@ -1,17 +1,19 @@
 ---
-title: "How to: Use a Dictionary to Store Event Instances - C# Programming Guide"
+title: "How to: use a dictionary to store event instances - C# Programming Guide"
 ms.custom: seodec18
-ms.date: 07/20/2015
+ms.date: 03/11/2019
 helpviewer_keywords: 
   - "events [C#], storing instances in a Dictionary"
 ms.assetid: 9512c64d-5aaf-40cd-b941-ca2a592f0064
 ---
-# How to: Use a Dictionary to Store Event Instances (C# Programming Guide)
-One use for `accessor-declarations` is to expose many events without allocating a field for each event, but instead using a Dictionary to store the event instances. This is only useful if you have many events, but you expect most of the events will not be implemented.  
-  
-## Example  
- [!code-csharp[csProgGuideEvents#9](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideEvents/CS/Events.cs#9)]  
-  
+# How to: use a dictionary to store event instances (C# Programming Guide)
+
+One use for the `add` and `remove` custom event accessors is to expose many events without allocating a field for each event, but instead using a <xref:System.Collections.Generic.Dictionary%602> instance to store the event instances, as the example below demonstrates. This is only useful if a type has many events, but you expect that most of the events will not be subscribed to.
```

Comment:
I suggest this addition:

> This implementation conforms to the behavior for [adding and removing delegates](../../../../_csharplang/spec/delegates#delegate-invocation) in the C# language specification.

Reply:
@BillWagner I've added the suggested sentence. Please let me know, if it's not fine to have it after the example, as it's now.